### PR TITLE
Some suggested clarifications and structure

### PR DESCRIPTION
**Pull request by gonuke**
Rather than leave a bunch of comments, I propose these improvements.

See a question in code comments about the deprecated material library format.

### CODE REVIEW

File: docs/pyapi/material_library.rst [hunk 1]
```diff
@@ -19,33 +19,40 @@ that is generally a semantically meaningful name, and the value is a :ref:`pyne_
...
-    /------/nucpath
-    /my_mat_metadata
+    /my_mat_lib/
+    /----------/nucpath
+    /my_mat_lib_metadata
     /nuclidelist
 
-`my_lib` (the `datapath` -- default `material`) is a HDF5 dataset containing the
+`my_mat_lib` (the `datapath` -- default `material`) is a HDF5 dataset containing the
 array of material composition, `nucpath` is a attribute containing the path to the
```
Comment (gonuke) [inline docs/pyapi/material_library.rst:53]: Is this all correct?  It seems too similar to the description for a `Material` and is written as if there is a single material, without really describing how multiple materials existed in this structure?

Comment (bam241) [reply docs/pyapi/material_library.rst:53]: yes basically all materials share the same `nuclidelist` and materials are appended to the datapath.

Comment (bam241) [reply docs/pyapi/material_library.rst:53]: it was built from Material, we just had the capability to add new material to it. (it worked properly as long as the new Materials didn't had extra nuclides...)

**Comment by bam241** [comment]
thx @gonuke 
merging

**Pull request merged**